src/SmartComponents/modules/reducers.js

```diff
@@ -153,10 +186,23 @@ function downloadCSV(driftData, systems) {
     link.dispatchEvent(new MouseEvent(`click`, { bubbles: true, cancelable: true, view: window }));
 }
 
+function toggleExpandedRow(expandedRows, factName) {
+    if (expandedRows.includes(factName)) {
+        expandedRows = _.remove(expandedRows, function(fact) {
```

Comment:
I think you can reverse this logic by using built in `filter` function and avoid needing lodash.